cgogn/geometry/algos/medial_axis.h

```diff
@@ -24,6 +24,7 @@
 #ifndef CGOGN_GEOMETRY_ALGOS_MEDIAL_AXIS_H_
 #define CGOGN_GEOMETRY_ALGOS_MEDIAL_AXIS_H_
 
+#include <cgogn/core/utils/numerics.h>
```

Comment:
Required on MSVC, contrarily to GCC where it's fine if it was included through another header before